charts/tekton-apps/values.yaml

```diff
@@ -184,8 +184,8 @@ runPostInstallMountPvcJob: false
 serviceAccount:
   # The name of the service account to use.
   name: "build-bot-sa"
-  # If we create several tekton-apps in the cluster we don't want to recreate the service account.
-  create: 'true'
+  # In which namespace service account is defined.
+  namespace: ""
```

Comment:
so is this SA used now I don't see it?
It looks like your eventlistener is running in a single SA defined in the tekton chart (build-bot-sa) and these atomic SAs are not being used for any pipeline execution?
https://tekton.dev/docs/triggers/triggers/

![image](https://github.com/user-attachments/assets/9c01dbf1-7c6d-43a0-b210-f8ec6c5c43ab)

Reply:
i.e. my question - the pipelinerun has to be executed under its own SA which is defined in the tekton-apps, but I can't find where it's being declared for concrete run.

Reply:
If we look at any Pipeline pod, we can see, that it actually uses that comonent-tailored SA, that has role grnating access only to secrets that belong to this project/component (ex. deploy key):

![image](https://github.com/user-attachments/assets/74fc9c99-5d5e-4af5-91d6-74591f54befa)

I believe, eihter EventListener or tekton-pipelines-controller calculate SA name in runtime from this, when PipelineRun is being created: 

![image](https://github.com/user-attachments/assets/263e869b-91cb-4163-a077-47d6897340db)